setup.py

```diff
@@ -90,6 +94,15 @@ def read(file_name):
 # update version information before launching setup
 update_version()
 
+DISTRO = platform.dist()[0]
+if DISTRO == "CentOS":
+    subprocess.run("sudo ./install.sh")
+elif DISTRO in ['debian', 'ubuntu']:
```

Comment:
elif DISTRO in ['Debian', 'Ubuntu']: since platform.dist() gives something like ('Ubuntu', '18.04', 'bionic')